### ISSUE TITLE
Electron 8, In AZERTY keyboard, global shortcut for CMD + A triggers the action for CMD + Q

### ISSUE THREAD

**Issue by jaga3421** [electron/electron#26125]
This is my simple code in index.js 
```
globalShortcut.register("CmdOrCtrl+q", () => {
            app.isQuiting = true;
            app.quit();
 })

```
This works fine. But when used via an azerty keyboard, CMD + A is triggering the actions inside this block.

### Issue Details

* **Electron Version:**
  * 8
* **Operating System:**
  * Mac 10.15.7

### Expected Behavior
CMD + A should select all 

### Actual Behavior
CMD + A is triggering events for CMD + Q

**Comment by ckerr** [comment]
Confirmed on Mac w / Electron 8, 9, 10.

https://gist.github.com/ckerr/26209ebe1c44d7c96811c344733ff854

Important note for maintainers not using an AZERTY layout: If you don't use an AZERTY keyboard layout natively, [change your Mac's keyboard input in System Settings and then switch to AZERTY in the keyboard indicator](https://www.macsx.com/how-to-change-your-macbooks-keyboard-layout-to-azerty/). This is a prerequisite to reproducing the issue.

**Comment by JoaquinScript** [comment]
Seems like this is a dup of [19747](https://github.com/electron/electron/issues/19747).

**Comment by jaga3421** [comment]
For time being, I have removed the global shortcuts and added local app shortcuts using Mousetrap library.

**Comment by Kilian** [comment]
There's a Fiddle and instructions available in #19747, so closing this as duplicate.
